src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -62,7 +62,11 @@ public boolean removeUser(User user) {
      * "equal" a user in the room at the application level, they are not the same at
      * the database level (different primary keys). This method therefore returns the
      * correct user object that's equal at both the application and database levels.
+     *
+     * @param userToFind the desired user to find
+     * @return the equivalent user in the database
      */
+
```

Comment:
nit: Remove line? Clearly you added that in, if there was a reason please lmk, but as I understood convent was to have no lines between the method comment and the method itself.

Reply:
Haha yep you're right, accidentally left a new line there